application/controllers/RoleController.php

```diff
@@ -143,6 +144,7 @@ public function removeAction()
 
     public function auditAction()
     {
+        $this->getTabs()->extend(new OutputFormat(["csv","json"]));
```

Comment:
Please add a space after commas in arrays.

Reply:
done

Reply:
I still see no space there.

```suggestion
        $this->getTabs()->extend(new OutputFormat(["csv", "json"]));
```